pkg/controller/vsphere/reconciler.go

```diff
@@ -333,6 +336,24 @@ func (r *Reconciler) reconcileNetwork(vm *virtualMachine) error {
 	return nil
 }
 
+func (r *Reconciler) reconcilePowerStateAnnontation(vm *virtualMachine) {
```

Comment:
can we please have a unit for this func?

Reply:
+1, would be good to see a couple of tests exercising this explicitly

Reply:
done